doc/releases/release-notes-3.7.rst

```diff
@@ -39,6 +39,8 @@ https://docs.zephyrproject.org/latest/security/vulnerabilities.html
 
 * CVE-2024-5931: Under embargo until 2024-09-10
 
+* CVE-2024-6135: Under embargo until 2024-09-11
```

Comment:
Why is 6137 not listed as well?

Reply:
not fixed yet.

Reply:
Ah right, of course! I thought this was the list of *new* CVEs for the release, should have read more carefully:)
Thanks!